core/validation/failures.py

```diff
@@ -511,15 +511,17 @@ def failures_to_messages(
     # one pre-transformation failure means payload is entirely pre-transformation failures
     if any(isinstance(failure, PreTransFormationFailure) for failure in validation_failures):
         # ignore tab and section for pre-transformation failures
+        # tab, section, message
         return {"PreTransformationErrors": [message for _, _, message in error_messages]}
```

Comment:
I feel like this could be named "pre_transformation_errors" Could you include this cahnge in your PR. Just for this return value from the API here.